examples/with-prerender/src/static_export.js

```diff
@@ -0,0 +1,25 @@
+import { renderStatic } from '@jaredpalmer/after';
+import afterRoutes from './routes';
+
+const assets = require(process.env.RAZZLE_ASSETS_MANIFEST);
+const chunks = require(process.env.RAZZLE_CHUNKS_MANIFEST);
+
+export const render = async (req, res) => {
+  try {
+    const { html, data } = await renderStatic({
```

Comment:
Is there no way for us keep the original render method for static? Our goal should be minimal amount of changes required

Reply:
Maybe do something like a ssgContext argument that after sets html and data on?

Reply:
Fixed all the issues now :)